docs/source/app-dev/authorization.rst

```diff
@@ -7,124 +7,195 @@ Authorization
+whether the request is valid according to the :ref:`Daml Ledger Model <da-ledgers>`.
+Whether a participant node *will* serve such a request to a Daml application depends on whether the
+request includes an access token that is valid and sufficient to authorize the request for this participant node.
 
-In particular, this model does not talk about authentication ("*Is the request claiming to come from Alice really sent by Alice?*")
-and authorization ("*Is Alice authorized to add a new Daml package to the ledger?*").
+Acquiring and using access tokens
+*********************************
 
-In practice, Daml ledgers will therefore need to add authentication to the ledger API.
-
-.. note::
-    Depending on the ledger topology, a Daml ledger may consist of multiple participant nodes,
-    each having its own ledger API server.
-    Each participant node typically hosts different Daml parties,
-    and only sees data visible to the parties hosted on that node (as defined by the Daml privacy model).
-
-    For more details on Daml ledger topologies, refer to the :ref:`Daml Ledger Topologies <daml-ledger-topologies>` documentation.
-
-Adding authentication
-=====================
-
-How authentication is set up on a particular ledger is defined by the ledger operator.
-However, most authentication setups share the following pattern:
+How an application should acquire access tokens depends on the participant node it talks to and is ultimately setup by the participant node operator.
+Many setups use a flow in the style of `OAuth 2.0 <https://oauth.net/2/>`_:
 
 First, the Daml application contacts a token issuer to get an access token.
-The token issuer verifies the identity of the requesting user
-(e.g., by checking the username/password credentials sent with the request),
-looks up the privileges of the user,
+The token issuer verifies the identity of the requesting application looks up the privileges of the application,
 and generates a signed access token describing those privileges.
 
-Then, the Daml application sends the access token along with each ledger API request.
-The Daml ledger verifies the signature of the token (to make sure it has not been tampered with),
-and then checks that the privileges described in the token authorize the given ledger API request.
+Then, the Daml application sends the access token along with every ledger API request.
+The Daml ledger verifies the signature of the token (to make sure it has not been tampered with and was issued by one of its trusted token issuers),
+and then checks that the token has not yet expired and that the privileges described in the token authorize the given ledger API request.
 
 .. image:: ./images/Authentication.svg
 
-Glossary:
+As shown above, using access tokens requires your application to attach them to every request.
+How to do that depends on the tool or library you use to interact with the ledger API.
```

Comment:
That is true, but I think we should describe the mechanism at the gRPC layer.
Ie something like "the Ledger API expects a trailer of the format here demonstrated with a gRPCurl command: "

Reply:
Will do in https://github.com/digital-asset/daml/issues/12501